src/SDKs/dirs.proj

```diff
@@ -47,6 +47,7 @@
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)StreamAnalytics\Management.StreamAnalytics\Microsoft.Azure.Management.StreamAnalytics.csproj" />
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)TrafficManager\Management.TrafficManager\Microsoft.Azure.Management.TrafficManager.csproj" />
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)WebSites\Management.Websites\Microsoft.Azure.Management.Websites.csproj" /> 
+		<ProjectToBuild Include="$(MSBuildThisFileDirectory)ApplicationInsights\Management.ApplicationInsights\Microsoft.Azure.Management.ApplicationInsights.csproj" /> 
```

Comment:
This is not needed anymore. This file should not be edited

Reply:
reverted the change. thanks!